modules/htlc/internal/keeper/keeper_test.go

```diff
@@ -40,9 +40,23 @@ type KeeperTestSuite struct {
 	app *simapp.SimApp
 }
 
+func (suite *KeeperTestSuite) SetupTest() {
+	app := simapp.Setup(false)
+
+	suite.cdc = app.Codec()
+	suite.ctx = app.BaseApp.NewContext(false, abci.Header{})
+	suite.app = app
+
+	initVars(suite)
+}
+
+func TestKeeperTestSuite(t *testing.T) {
+	suite.Run(t, new(KeeperTestSuite))
+}
+
 func initVars(suite *KeeperTestSuite) {
-	addrSender = sdk.AccAddress([]byte("__addrSender________"))
-	addrTo = sdk.AccAddress([]byte("__addrTo____________"))
+	addrSender = sdk.AccAddress("__addrSender________")
```

Comment:
you can use tendermint/crypto.AddressHash([]byte("string")) to generate the addr.